src/OpenTracing/Exceptions/InvalidReferenceArgument.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace OpenTracing\Exceptions;
+
+use InvalidArgumentException;
+
+final class InvalidReferenceArgument extends InvalidArgumentException
+{
+    public static function emptyType()
```

Comment:
I don't understand the disadvantage of having two classes for these reasons that inherit from `InvalidArgumentException` and set a default message, but don't want to bike shed if you feel strongly about this :)